cfgrammar/src/lib/yacc/grammar.rs

```diff
@@ -181,6 +187,18 @@ impl YaccGrammar<u32> {
     }
 }
 
+impl<StorageT: 'static + PrimInt + Unsigned> FromStr for YaccGrammar<StorageT>
+where
+    usize: AsPrimitive<StorageT>,
+{
+    type Err = Vec<YaccGrammarError>;
+    fn from_str(s: &str) -> YaccGrammarResult<Self> {
+        let ast_validation = ast::ASTWithValidityInfo::from_str(s)
+            .map_err(|errs| errs.iter().cloned().map(|e| e.into()).collect::<Vec<_>>())?;
```

Comment:
On my walk it occurred to me that it'd be better to use `drain` here over `cloned`, for some reason I thought there was another instance of this, which would allow us to remove `T: Clone` bounds, but I'm not seeing it right this moment.

Reply:
Should be fixed in [edf677f](https://github.com/softdevteam/grmtools/pull/554/commits/edf677fcba9a09dc60d4ef64257b7e370b5b49a8)

Must have been imagining the other `cloned()` call, because